.github/workflows/build.yaml

```
@@ -0,0 +1,33 @@
+name: build
+
+on:
+  push:
+    branches:
+      - "main"
+    tags:
+      - v[0-9][0-9].[0-9][0-9].[0-9][0-9]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  conda-build:
+    needs: check-style
+    runs-on: ubuntu-latest
+    container:
+      image: rapidsai/ci-conda:latest
+    steps:
+      - uses: actions/checkout@v3
```

We can bump this to `v4` if we don't need to build on CentOS 7.
```suggestion
      - uses: actions/checkout@v4
```